compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -1030,7 +1030,7 @@ std::optional<int> getGPUSubgroupSize(mlir::FunctionOpInterface func) {
 
 SmallVector<IREE::HAL::ExecutableVariantOp>
 getExecutableVariantOps(mlir::ModuleOp moduleOp) {
-  llvm::SmallVector<IREE::HAL::ExecutableVariantOp> executableVariantOps;
+  SmallVector<IREE::HAL::ExecutableVariantOp> executableVariantOps;
```

Comment:
This looks like an old change?

Reply:
This is not resolved. If these are old changes, we should rebase this PR to be reasonably up-to-date with main.